common/src/main/java/com/wynntils/gui/render/MapRenderer.java

```diff
@@ -335,7 +334,7 @@ public static void renderTexturedPoi(
                 renderablePoi.getIcon().resource(),
                 renderX,
                 renderZ,
-                renderablePoi.getLocation().getY(),
+                0,
```

Comment:
What was the original purpose, and how does jumping break things?

Reply:
No idea, and this is not needed as we sort the waypoint rendering beforehand.